### PR TITLE
Theme handling in element

### PR DESCRIPTION
**Pull request by gabrielonilsson**

### CODE REVIEW

File: src/components/element/index.tsx [hunk 1]
```diff
@@ -15,11 +15,19 @@ export class Form {
 	@Prop() apiKey: string
 	@Prop({ reflectToAttr: true, mutable: true }) state: "failed" | "succeeded" | "processing" | "created" = "created"
 	@Prop({ mutable: true }) value?: AuthorizationCreatableSafe
+	private themeName?: string
```
Comment (simonmika) [inline src/components/element/index.tsx]: Should this be a `@State()`?

Comment (simonmika) [reply src/components/element/index.tsx]: And just call it `theme` for simplicity.

**Review by fredrikbryntesson — DISMISSED** [review]

File: src/components/element/index.tsx [hunk 2]
```diff
@@ -15,11 +15,19 @@ export class Form {
...
 	@Event() changed: EventEmitter<Authorization>
 	private received?: (state: "succeeded" | "failed", authorization: Authorization | Token) => void
 	@State() payload?: Payload
 	componentWillLoad() {
 		new Verifier("public").verify(this.apiKey).then(payload => this.payload = payload)
+		const styleLinks = document.querySelectorAll("link[rel=stylesheet]") as NodeListOf<HTMLLinkElement>
```
Comment (simonmika) [inline src/components/element/index.tsx]: `"link[rel=stylesheet][href^=https://theme.payfunc.com/]"`

**Review by simonmika — CHANGES_REQUESTED** [review]

File: src/components/element/index.tsx (hunk 2 above)
Comment (simonmika) [reply src/components/element/index.tsx]: Maybe even using `querySelector` instead when you only query links to our themes.

**Review by simonmika — APPROVED** [review]

**Pull request merged**